### PR TITLE
improve run.sh script for evaluation.

### PR DESCRIPTION
**Pull request by Error323**

### CODE REVIEW

**Comment by alg4032** [comment]
Can I ask some questions about the match conditions?
1) Is the "playouts" setting comparable to the "nodestime" setting in Stockfish? I'm asking because in my understanding two different engines should play at the same **time** conditions in match, not depth. Even a "nodestime" number would be inappropiate for two Stockfish versions in a match unless in specific cases where the differences between two engine versions are minor.
2) Is it planned to switch to SPRT for evaluation of match results at some point? I guess in general a higher number of games than 100 will be required too once the progress rate begins to shrink.

**Comment by jkiliani** [comment]
Playouts essentially means neural net evaluations, which take the bulk of the computing power. Matches between engines with the same neural net architecture will use very close to equal time, and there are good reasons for not using fixed time controls instead: The number of playouts per second is highly dependent on hardware, and different setups should be testing networks at same search strength, not the same time settings.

File: scripts/run.sh [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+NXT=lc_gen2
+CUR=lc_gen1
+
+cutechess-cli -rounds 50 -tournament gauntlet -concurrency 4 \
  -pgnout results.pgn \
- -engine name=lc_new cmd=$DIR/lczero arg="-n" arg="-t1" arg="--weights=$DIR/newweights.txt" \
- -engine name=lc_base cmd=$DIR/lczero arg="-n" arg="-t1" arg="--weights=$DIR/weights.txt" \
- -each proto=uci tc=inf
+ -engine name=$NXT cmd=$DIR/lczero arg="--threads=1" arg="--noise" arg="--weights=$WDR/gen2-64x6.txt" arg="--playouts=800" arg="--noponder" arg="--gpu=0" tc=inf \
+ -engine name=$CUR cmd=$DIR/lczero arg="--threads=1" arg="--noise" arg="--weights=$WDR/seed-64x6.txt" arg="--playouts=800" arg="--noponder" arg="--gpu=1" tc=inf \
```
Comment (glinscott) [inline scripts/run.sh]: Super small thing - why not leave the "tc=inf" in the "each" below?

Comment (Error323) [reply scripts/run.sh]: Oh that's a remminent from playing gnuchess. I'll fix it when at my computer again.

**Pull request merged**